src/Localization/en/translations.json

```diff
@@ -37,6 +37,46 @@
             "OnSaveFailureMessage": "Data save operation not currently permitted."
         },
         "SqlX": {
+            "DedicatedGatewayDescription": "Provision dedicated gateways for Azure CosmosDB accounts. Dedicated gateways may improve performance in certain scenarios. For details, ",
+            "DedicatedGateway": "Dedicated Gateway",
+            "Enable": "Enable",
+            "Disable": "Disable",
+            "LearnAboutDedicatedGateway": "read more about dedicated gateway.",
+            "SKUs": "SKUs",
+            "SKUsPlaceHolder": "Select SKUs",
+            "NumberOfInstances": "Number of instances",
+            "CosmosD4s": "Cosmos.D4s (General Purpose Cosmos Compute with 4 vCPUs, 16 GB Memory)",
+            "CosmosD8s": "Cosmos.D8s (General Purpose Cosmos Compute with 8 vCPUs, 32 GB Memory)",
+            "CosmosD16s": "Cosmos.D16s (General Purpose Cosmos Compute with 16 vCPUs, 64 GB Memory)",
+            "CosmosD32s": "Cosmos.D32s (General Purpose Cosmos Compute with 32 vCPUs, 128 GB Memory)",
+            "CreateMessage": "DedicatedGateway resource is being created.",
```

Comment:
Need to finalize these strings

Reply:
Let's keep these strings, these are correct for now!